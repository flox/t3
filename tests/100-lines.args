-- sh -c 'for i in `seq 1 100`; do echo line $i to stdout; sleep 0.01; echo line $i to stderr 1>&2; sleep 0.01; done'
