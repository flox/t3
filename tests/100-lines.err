line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 11 to stderr
line 12 to stderr
line 13 to stderr
line 14 to stderr
line 15 to stderr
line 16 to stderr
line 17 to stderr
line 18 to stderr
line 19 to stderr
line 20 to stderr
line 21 to stderr
line 22 to stderr
line 23 to stderr
line 24 to stderr
line 25 to stderr
line 26 to stderr
line 27 to stderr
line 28 to stderr
line 29 to stderr
line 30 to stderr
line 31 to stderr
line 32 to stderr
line 33 to stderr
line 34 to stderr
line 35 to stderr
line 36 to stderr
line 37 to stderr
line 38 to stderr
line 39 to stderr
line 40 to stderr
line 41 to stderr
line 42 to stderr
line 43 to stderr
line 44 to stderr
line 45 to stderr
line 46 to stderr
line 47 to stderr
line 48 to stderr
line 49 to stderr
line 50 to stderr
line 51 to stderr
line 52 to stderr
line 53 to stderr
line 54 to stderr
line 55 to stderr
line 56 to stderr
line 57 to stderr
line 58 to stderr
line 59 to stderr
line 60 to stderr
line 61 to stderr
line 62 to stderr
line 63 to stderr
line 64 to stderr
line 65 to stderr
line 66 to stderr
line 67 to stderr
line 68 to stderr
line 69 to stderr
line 70 to stderr
line 71 to stderr
line 72 to stderr
line 73 to stderr
line 74 to stderr
line 75 to stderr
line 76 to stderr
line 77 to stderr
line 78 to stderr
line 79 to stderr
line 80 to stderr
line 81 to stderr
line 82 to stderr
line 83 to stderr
line 84 to stderr
line 85 to stderr
line 86 to stderr
line 87 to stderr
line 88 to stderr
line 89 to stderr
line 90 to stderr
line 91 to stderr
line 92 to stderr
line 93 to stderr
line 94 to stderr
line 95 to stderr
line 96 to stderr
line 97 to stderr
line 98 to stderr
line 99 to stderr
line 100 to stderr
