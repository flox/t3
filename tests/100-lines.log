[38;5;141m[0mline 1 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0mline 2 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0mline 3 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0mline 4 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0mline 5 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0mline 6 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0mline 7 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0mline 8 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0mline 9 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0mline 10 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0mline 11 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 11 to stderr[0m
[38;5;141m[0mline 12 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 12 to stderr[0m
[38;5;141m[0mline 13 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 13 to stderr[0m
[38;5;141m[0mline 14 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 14 to stderr[0m
[38;5;141m[0mline 15 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 15 to stderr[0m
[38;5;141m[0mline 16 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 16 to stderr[0m
[38;5;141m[0mline 17 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 17 to stderr[0m
[38;5;141m[0mline 18 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 18 to stderr[0m
[38;5;141m[0mline 19 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 19 to stderr[0m
[38;5;141m[0mline 20 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 20 to stderr[0m
[38;5;141m[0mline 21 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 21 to stderr[0m
[38;5;141m[0mline 22 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 22 to stderr[0m
[38;5;141m[0mline 23 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 23 to stderr[0m
[38;5;141m[0mline 24 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 24 to stderr[0m
[38;5;141m[0mline 25 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 25 to stderr[0m
[38;5;141m[0mline 26 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 26 to stderr[0m
[38;5;141m[0mline 27 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 27 to stderr[0m
[38;5;141m[0mline 28 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 28 to stderr[0m
[38;5;141m[0mline 29 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 29 to stderr[0m
[38;5;141m[0mline 30 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 30 to stderr[0m
[38;5;141m[0mline 31 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 31 to stderr[0m
[38;5;141m[0mline 32 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 32 to stderr[0m
[38;5;141m[0mline 33 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 33 to stderr[0m
[38;5;141m[0mline 34 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 34 to stderr[0m
[38;5;141m[0mline 35 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 35 to stderr[0m
[38;5;141m[0mline 36 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 36 to stderr[0m
[38;5;141m[0mline 37 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 37 to stderr[0m
[38;5;141m[0mline 38 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 38 to stderr[0m
[38;5;141m[0mline 39 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 39 to stderr[0m
[38;5;141m[0mline 40 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 40 to stderr[0m
[38;5;141m[0mline 41 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 41 to stderr[0m
[38;5;141m[0mline 42 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 42 to stderr[0m
[38;5;141m[0mline 43 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 43 to stderr[0m
[38;5;141m[0mline 44 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 44 to stderr[0m
[38;5;141m[0mline 45 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 45 to stderr[0m
[38;5;141m[0mline 46 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 46 to stderr[0m
[38;5;141m[0mline 47 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 47 to stderr[0m
[38;5;141m[0mline 48 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 48 to stderr[0m
[38;5;141m[0mline 49 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 49 to stderr[0m
[38;5;141m[0mline 50 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 50 to stderr[0m
[38;5;141m[0mline 51 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 51 to stderr[0m
[38;5;141m[0mline 52 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 52 to stderr[0m
[38;5;141m[0mline 53 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 53 to stderr[0m
[38;5;141m[0mline 54 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 54 to stderr[0m
[38;5;141m[0mline 55 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 55 to stderr[0m
[38;5;141m[0mline 56 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 56 to stderr[0m
[38;5;141m[0mline 57 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 57 to stderr[0m
[38;5;141m[0mline 58 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 58 to stderr[0m
[38;5;141m[0mline 59 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 59 to stderr[0m
[38;5;141m[0mline 60 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 60 to stderr[0m
[38;5;141m[0mline 61 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 61 to stderr[0m
[38;5;141m[0mline 62 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 62 to stderr[0m
[38;5;141m[0mline 63 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 63 to stderr[0m
[38;5;141m[0mline 64 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 64 to stderr[0m
[38;5;141m[0mline 65 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 65 to stderr[0m
[38;5;141m[0mline 66 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 66 to stderr[0m
[38;5;141m[0mline 67 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 67 to stderr[0m
[38;5;141m[0mline 68 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 68 to stderr[0m
[38;5;141m[0mline 69 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 69 to stderr[0m
[38;5;141m[0mline 70 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 70 to stderr[0m
[38;5;141m[0mline 71 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 71 to stderr[0m
[38;5;141m[0mline 72 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 72 to stderr[0m
[38;5;141m[0mline 73 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 73 to stderr[0m
[38;5;141m[0mline 74 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 74 to stderr[0m
[38;5;141m[0mline 75 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 75 to stderr[0m
[38;5;141m[0mline 76 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 76 to stderr[0m
[38;5;141m[0mline 77 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 77 to stderr[0m
[38;5;141m[0mline 78 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 78 to stderr[0m
[38;5;141m[0mline 79 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 79 to stderr[0m
[38;5;141m[0mline 80 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 80 to stderr[0m
[38;5;141m[0mline 81 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 81 to stderr[0m
[38;5;141m[0mline 82 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 82 to stderr[0m
[38;5;141m[0mline 83 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 83 to stderr[0m
[38;5;141m[0mline 84 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 84 to stderr[0m
[38;5;141m[0mline 85 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 85 to stderr[0m
[38;5;141m[0mline 86 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 86 to stderr[0m
[38;5;141m[0mline 87 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 87 to stderr[0m
[38;5;141m[0mline 88 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 88 to stderr[0m
[38;5;141m[0mline 89 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 89 to stderr[0m
[38;5;141m[0mline 90 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 90 to stderr[0m
[38;5;141m[0mline 91 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 91 to stderr[0m
[38;5;141m[0mline 92 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 92 to stderr[0m
[38;5;141m[0mline 93 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 93 to stderr[0m
[38;5;141m[0mline 94 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 94 to stderr[0m
[38;5;141m[0mline 95 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 95 to stderr[0m
[38;5;141m[0mline 96 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 96 to stderr[0m
[38;5;141m[0mline 97 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 97 to stderr[0m
[38;5;141m[0mline 98 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 98 to stderr[0m
[38;5;141m[0mline 99 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 99 to stderr[0m
[38;5;141m[0mline 100 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 100 to stderr[0m
