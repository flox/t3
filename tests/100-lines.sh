#!/bin/sh
./t3 "$1" -- sh -c 'for i in `seq 1 100`; do echo line $i to stdout; sleep 0; echo line $i to stderr 1>&2; sleep 0; done'
