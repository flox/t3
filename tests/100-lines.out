line 1 to stdout
line 2 to stdout
line 3 to stdout
line 4 to stdout
line 5 to stdout
line 6 to stdout
line 7 to stdout
line 8 to stdout
line 9 to stdout
line 10 to stdout
line 11 to stdout
line 12 to stdout
line 13 to stdout
line 14 to stdout
line 15 to stdout
line 16 to stdout
line 17 to stdout
line 18 to stdout
line 19 to stdout
line 20 to stdout
line 21 to stdout
line 22 to stdout
line 23 to stdout
line 24 to stdout
line 25 to stdout
line 26 to stdout
line 27 to stdout
line 28 to stdout
line 29 to stdout
line 30 to stdout
line 31 to stdout
line 32 to stdout
line 33 to stdout
line 34 to stdout
line 35 to stdout
line 36 to stdout
line 37 to stdout
line 38 to stdout
line 39 to stdout
line 40 to stdout
line 41 to stdout
line 42 to stdout
line 43 to stdout
line 44 to stdout
line 45 to stdout
line 46 to stdout
line 47 to stdout
line 48 to stdout
line 49 to stdout
line 50 to stdout
line 51 to stdout
line 52 to stdout
line 53 to stdout
line 54 to stdout
line 55 to stdout
line 56 to stdout
line 57 to stdout
line 58 to stdout
line 59 to stdout
line 60 to stdout
line 61 to stdout
line 62 to stdout
line 63 to stdout
line 64 to stdout
line 65 to stdout
line 66 to stdout
line 67 to stdout
line 68 to stdout
line 69 to stdout
line 70 to stdout
line 71 to stdout
line 72 to stdout
line 73 to stdout
line 74 to stdout
line 75 to stdout
line 76 to stdout
line 77 to stdout
line 78 to stdout
line 79 to stdout
line 80 to stdout
line 81 to stdout
line 82 to stdout
line 83 to stdout
line 84 to stdout
line 85 to stdout
line 86 to stdout
line 87 to stdout
line 88 to stdout
line 89 to stdout
line 90 to stdout
line 91 to stdout
line 92 to stdout
line 93 to stdout
line 94 to stdout
line 95 to stdout
line 96 to stdout
line 97 to stdout
line 98 to stdout
line 99 to stdout
line 100 to stdout
